src/InputGuesser.test.tsx

```diff
@@ -209,11 +209,8 @@ describe('<InputGuesser />', () => {
       'resources.users.fields.embedded',
     );
     expect(embeddedField).toHaveValue('{"address":"91 rue du Temple"}');
-    expect(
-      await screen.findAllByText('resources.users.fields.embeddeds.0'),
-    ).toHaveLength(1);
-    const embeddedsField = screen.getByLabelText(
-      'resources.users.fields.embeddeds.0',
+    const embeddedsField = await screen.findByLabelText(
+      'resources.users.fields.embeddeds',
```

Comment:
This is due to a change in RA v5: ArrayInputs with scalar value no longer have a label of the form `resources.users.fields.embeddeds.0`. The recommendation is now to provide your own label (which I did).